packages/ag-charts-community/src/chart/cartesianChart.ts

```diff
@@ -442,6 +451,30 @@ export class CartesianChart extends Chart {
         return { clipSeries, axisThickness };
     }
 
+    private getSyncedDomain(axis: ChartAxis) {
+        const syncModule = this.modulesManager.getModule<SyncModule>('sync');
+        if (!syncModule?.enabled) return;
```

Comment:
Just to confirm, if we short-circuit here, then the axis/domain behaviour for non-synchonised charts is as before?

Reply:
Yes